src/main/java/com/systemmeltdown/robotlib/CanIdMap.java

```diff
@@ -28,4 +28,8 @@
    * Mechanisms and other devices (robot specific)
    */
 
+  public static final int[] LEFT_ENCODER_PORTS = new int[]{11, 13};
```

Comment:
Are these the only encoders on the left and right? I think this might need to be more specific.